index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href='styles.css'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Challenge</title>
+  </head>
```

Comment:
- [ ] Good job for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and you phrased the comments politely for suggested changes,  But Kindly makes sure that ,you  includes an explanation, a "why", for each requested change.

Ex. "I think it would be better to link .Css file with your HTML code,  because the .CSS define styles for your web pages, including the design, layout and variations in display for different devices and screen sizes".